### PR TITLE
Create UI movie detail screen

### PR DESCRIPTION
**Pull request by nvquangth**
Function: code UI movie detail screen
Redmine: https://edu-redmine.framgia.vn/issues/1307

### CODE REVIEW

**Comment by nvquangth** [comment]
ready

File: app/src/main/res/layout/activity_movie_detail.xml [hunk 1]
```diff
@@ -1,8 +1,199 @@
...
+                android:layout_marginStart="@dimen/dp_8"
+                android:layout_marginTop="@dimen/dp_8"
+                android:orientation="horizontal"
+                app:layoutManager="android.support.v7.widget.LinearLayoutManager"
+                app:layout_constraintEnd_toEndOf="parent"
+                app:layout_constraintStart_toStartOf="parent"
+                app:layout_constraintTop_toBottomOf="@+id/textView10" />
+
+            <TextView
+                android:id="@+id/textView10"
```
Comment (tuannguyen38) [inline app/src/main/res/layout/activity_movie_detail.xml]: sao lại textView10. Convention 3.2

File: app/src/main/res/layout/activity_movie_detail.xml [hunk 2]
```diff
@@ -1,8 +1,199 @@
...
+                android:layout_marginStart="@dimen/dp_8"
+                android:layout_marginTop="@dimen/dp_8"
+                android:nestedScrollingEnabled="false"
+                app:layoutManager="android.support.v7.widget.LinearLayoutManager"
+                app:layout_constraintEnd_toEndOf="parent"
+                app:layout_constraintStart_toStartOf="parent"
+                app:layout_constraintTop_toBottomOf="@+id/textView8" />
+
+            <TextView
+                android:id="@+id/textView8"
```
Comment (tuannguyen38) [inline app/src/main/res/layout/activity_movie_detail.xml]: tương tự

File: app/src/main/res/layout/activity_movie_detail.xml [hunk 3]
```diff
@@ -1,8 +1,199 @@
...
+                android:layout_marginRight="@dimen/dp_8"
+                android:layout_marginStart="@dimen/dp_8"
+                android:layout_marginTop="@dimen/dp_8"
+                android:orientation="horizontal"
+                app:layoutManager="android.support.v7.widget.LinearLayoutManager"
+                app:layout_constraintEnd_toEndOf="parent"
+                app:layout_constraintStart_toStartOf="parent"
+                app:layout_constraintTop_toBottomOf="@+id/textView4" />
+
+            <TextView
```
Comment (tuannguyen38) [inline app/src/main/res/layout/activity_movie_detail.xml]: như trên

**Comment by nvquangth** [comment]
ready

**Pull request merged**